src/libtools/akimaspline.cc

```diff
@@ -141,7 +136,7 @@ void AkimaSpline::Interpolate(ub::vector<double> &x, ub::vector<double> &y)
     }
 }
 
-void AkimaSpline::Fit(ub::vector<double> &x, ub::vector<double> &y)
+void AkimaSpline::Fit(Eigen::VectorXd &x, Eigen::VectorXd &y)
```

Comment:
Why do we even have this function?

Reply:
akima must implement this because it is implementing a virtual class